environment.yml

```diff
@@ -9,5 +9,10 @@ dependencies:
 - cookiecutter
 - pytest
 - pytest-cookies
-- bumpversion
+- pytest-notebook
+- bump2version
 - watchdog
+- ipython
+- nbsphinx
+- nbval
+- nbconvert
```

Comment:
When I try to create an environment from this, it seems to stall. Does this also happen on your end ?

Reply:
These changes should go into the cookie-cutter template, no?

Reply:
Woops! Yes, you're right!